### ISSUE TITLE
shorten JVM waiting time

### ISSUE THREAD

**Issue by TobiasGoerke** [viadee/anchorsOnR#11]
In `R/connections.R` ->` initAnchors()` the JVM is started. Til now, the function just falls asleep for 5 seconds to let the JVM start. Clearly not a good solution. We need to find a dynamic way to handle this.

```
.anchors.startJar(ip = ip, port = port, name = name, ice_root = tempdir(), stdout = stdout, bind_to_localhost = FALSE, log_dir = NA, log_level = NA, context_path = NA)

Sys.sleep(5L)
```

**Comment by TobiasGoerke** [comment]
Related to #15

**Comment by TobiasGoerke** [comment]
Closed by 7b2fc0369a92d439e1976cfd3948c836f59ee959
